.storybook/main.js

```diff
@@ -74,5 +78,7 @@ function overrideDefaultBabelLoader(rules) {
     throw new Error('storybook webpack rules changed');
   }
 
-  loader.options.customize = customLoaderPath;
+  if (loader) {
```

Comment:
hmm why is this check needed ?

Reply:
Good catch! This was a change that I made earlier to make things work before I knew that the `rules` object changed shaped (the `loader` became a path string and not just a name string) which I addressed in this [commit ](https://github.com/microsoft/fluentui/pull/18693/commits/88021315a5e0a4551ab8664db5e374082523c8ae)- I just forgot to remove this line 😅.